src/components/BuildStatusIcon/BuildStatusIcon.tsx

```diff
@@ -51,16 +52,20 @@ export const BuildStatusIcon = ({ build, long }: IBuildStatusIcon) => {
     ((build as Build).attributes?.POST_BUILD_REPO_VALIDATION === 'REPO_SYSTEM_ERROR' ||
       (build as Build).attributes?.PNC_SYSTEM_ERROR === 'DISABLED_FIREWALL');
 
+  const buildLogLink = selectedIconData.failed ? `/builds/${build.id}/alignment-log` : `/builds/${build.id}/build-log`;
```

Comment:
@jomrazek is this condition valid?

Reply:
probably some other condition would suit better, I put there some just as proof of concept

Reply:
We discussed another approach on PM.